webpack/scenes/ContentViews/ContentViewsActions.js

```diff
@@ -1,6 +1,8 @@
-import { API_OPERATIONS, get } from 'foremanReact/redux/API';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { API_OPERATIONS, get, post } from 'foremanReact/redux/API';
```

Comment:
Check out https://github.com/theforeman/foreman/pull/8153

Now that this has been merged, it may simplify our API-handling code even more!
(great work @amirfefer)